.pre-commit-hooks.yaml

```diff
@@ -4,8 +4,16 @@
   language: docker_image
   entry: returntocorp/semgrep:develop
 
-- id: semgrep
+- id: semgrep-docker
   name: semgrep
   description: This hook runs semgrep
   language: docker_image
   entry: returntocorp/semgrep:latest
+
+- id: semgrep
+  name: semgrep
+  entry: semgrep
+  language: python
+  args: []
```

Comment:
It looks like these are the default values for `args`, `require_serial`, and `additional_dependencies`, so we shouldn't need to re-specify them here.